src/components/shared/Button.js

```diff
@@ -7,6 +8,18 @@ import styled, { css } from 'styled-components'
 */
 
 const CustomButton = styled(GrommetButton)`
+  ${({ uppercase }) =>
+    uppercase &&
+    css`
+      text-transform: uppercase;
+    `}
+
+  ${({ large, viewport }) =>
+    large &&
+    css`
+      font-size: ${viewport !== 'small' ? '20px' : '16px'};
+      padding: ${viewport !== 'small' ? '16px 32px' : '4px 24px'};
```

Comment:
this can be added to the button directly using the currently named setResponsive function

```js
<GrommetButton
  pad={setResponsive({
    vertical: "xxsmall",  
    horizontal: "medium"
  }, {
    vertical: "small", 
    horizontal: "large"
}}
```

Reply:
🗒️ The `pad` was removed from the styled-component rule and added using the `setResponsive`